modules/date.js

```diff
@@ -0,0 +1,8 @@
+let luxon;
+const date = luxon.DateTime.local().toLocaleString(luxon.DateTime.DATE_MED);
+const { hour } = luxon.DateTime.local().c;
+const min = luxon.DateTime.local().c.minute;
+const sec = luxon.DateTime.local().c.second;
+export {
+  date, hour, min, sec,
```

Comment:
- [ ] Great job working on the luxon library but also I can't see any time displayed beside the nav as required in the requirements so please solve the issue here to just display the time as required 🤩✨🔥💯💯

![image](https://user-images.githubusercontent.com/58553711/220233705-18bb8144-f07a-4ea1-a233-e9b517db5801.png)

Reply:
> * [ ]  Great job working on the luxon library but also I can't see any time displayed beside the nav as required in the requirements so please solve the issue here to just display the time as required star_strucksparklesfire100100
> 
> ![image](https://user-images.githubusercontent.com/58553711/220233705-18bb8144-f07a-4ea1-a233-e9b517db5801.png)

- [ ] As requested by the previous reviewer above ☝🏻 Let's make sure to displaye the current date below the navigation bar, here is your curent design 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/220240420-77fd2ca5-17ea-47dc-86db-732a39bdb68c.png)

Reply:
Are you blind motherfucker?